cli/tests/common/test_environment.rs

```diff
@@ -143,6 +143,14 @@ impl TestEnvironment {
         cmd.env("JJ_OP_TIMESTAMP", timestamp.to_rfc3339());
 
         if cfg!(windows) {
+            cmd.env(
+                "APPDATA",
+                self.home_dir
+                    .join("Appdata")
+                    .join("Roaming")
+                    .to_str()
+                    .unwrap(),
+            );
```

Comment:
nit: probably should move to a separate commit with its own explanation of why it was done